tests/evhtests/l7_evh_crd_test.go

```diff
@@ -2414,3 +2449,417 @@ func TestDeleteHostRuleWithActiveL7Rule(t *testing.T) {
 	TearDownIngressForCacheSyncCheck(t, modelName)
 
 }
+
+func TestHostRuleUseRegexSecure(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	modelName, _ := GetModelName("foo.com", "default")
+	hrname := "samplehr-foo"
+	fqdn := "foo.com"
+	namespace := "default"
+
+	SetUpIngressForCacheSyncCheck(t, true, true, modelName)
+
+	hostrule := integrationtest.FakeHostRule{
+		Name:      hrname,
+		Namespace: namespace,
+		Fqdn:      fqdn,
+		UseRegex:  true,
+	}
+	hrCreate := hostrule.HostRule()
+	if _, err := lib.AKOControlConfig().V1beta1CRDClientset().AkoV1beta1().HostRules(namespace).Create(context.TODO(), hrCreate, metav1.CreateOptions{}); err != nil {
+		t.Fatalf("error in adding HostRule: %v", err)
+	}
+	g.Eventually(func() string {
+		hostrule, _ := v1beta1CRDClient.AkoV1beta1().HostRules("default").Get(context.TODO(), hrname, metav1.GetOptions{})
+		return hostrule.Status.Status
+	}, 20*time.Second).Should(gomega.Equal("Accepted"))
+
+	sniVSKey := cache.NamespaceName{Namespace: "admin", Name: lib.Encode("cluster--foo.com", lib.EVHVS)}
```

Comment:
might want to change it to evhVSKey

Reply:
This name **sniVSKey** is used all throughout the suite so it will be better to change it at a later time.